confluent_server/confluent/messages.py

```diff
@@ -1023,6 +1024,21 @@ def __init__(self, name=None, dn=None):
             self.kvpairs = {name: kv}
 
 
+class License(ConfluentMessage):
+    readonly = True
+
+    def __init__(self, name=None, kvm=None):
+        self.notnode = name is None
+        self.desc = 'License'
+
+        kv = []
+        kv.append({'Remote KVM availability': str(kvm)})
```

Comment:
I think I'd rather have lowercase and no spaces in the keynames, to be consistent with other things.